tsl/src/compression/create.c

```diff
@@ -479,11 +479,14 @@ create_compressed_table_indexes(Oid compresstable_relid, CompressColInfo *compre
 							 InvalidOid, /* IndexRelationId */
 							 InvalidOid, /* parentIndexId */
 							 InvalidOid, /* parentConstraintId */
-							 false,		 /* is_alter_table */
-							 false,		 /* check_rights */
-							 false,		 /* check_not_in_use */
-							 false,		 /* skip_build */
-							 false);	 /* quiet */
+#if PG16_GE
+							 -1, /* total_parts */
+#endif
+							 false,	 /* is_alter_table */
+							 false,	 /* check_rights */
+							 false,	 /* check_not_in_use */
```

Comment:
Let's add a compat macro for this function as well? I see at least three usages.

Reply:
Done.